Shared/mods/deathmatch/logic/luascripts/constants_other.lua.h

```diff
@@ -0,0 +1,378 @@
+    V = 'v',
+    W = 'w',
+    X = 'x',
+    Y = 'y',
+    Z = 'z',
+    Num0 = 'num_0',
+    Num1 = 'num_1',
+    Num2 = 'num_2',
+    Num3 = 'num_3',
+    Num4 = 'num_4',
+    Num5 = 'num_5',
+    Num6 = 'num_6',
+    Num7 = 'num_7',
+    Num8 = 'num_8',
+    Num9 = 'num_9',
+    NumMul = 'num_mul',
+    NumAdd = 'num_add',
+    NumSep = 'num_sep',
+    NumSub = 'num_sub',
+    NumDiv = 'num_div',
+    NumDec = 'num_dec',
+    NumEnter = 'num_enter',
+    F1 = 'F1',
+    F2 = 'F2',
+    F3 = 'F3',
+    F4 = 'F4',
+    F5 = 'F5',
+    F6 = 'F6',
+    F7 = 'F7',
+    F8 = 'F8',
+    F9 = 'F9',
+    F10 = 'F10',
+    F11 = 'F11',
+    F12 = 'F12',
+    Escape = 'escape',
+    Backspace = 'backspace',
+    Tab = 'tab',
+    LeftAlt = 'lalt',
+    RightAlt = 'ralt',
+    Enter = 'enter',
+    Space = 'space',
+    PageUp = 'pgup',
+    PageDown = 'pgdn',
+    End = 'end',
+    Home = 'home',
+    Insert = 'insert',
+    Delete = 'delete',
+    LeftShift = 'lshift',
+    RightShift = 'rshift',
+    LeftCtrl = 'lctrl',
+    RightCtrl = 'rctrl',
+    LeftSquareBracket = '[',
+    RightSquareBracket = ']',
+    Pause = 'pause',
+    Capslock = 'capslock',
+    Scroll = 'scroll',
+    Semicolon = ';',
+    Comma = ',',
+    Dash = '-',
+    Dot = '.',
+    Slash = '/',
+    Hash = '#',
+    BackSlash = '\\',
+    Equals = '='
+}
+local _controlNames = {
+    Fire = 'fire',
+    AimWeapon = 'aim_weapon',
+    NextWeapon = 'next_weapon',
+    PreviousWeapon = 'previous_weapon',
+    Forwards = 'forwards',
+    Backwards = 'backwards',
+    Left = 'left',
+    Right = 'right',
+    ZoomIn = 'zoom_in',
+    ZoomOut = 'zoom_out',
+    ChangeCamera = 'change_camera',
+    Jump = 'jump',
+    Sprint = 'sprint',
+    LookBehind = 'look_behind',
+    Crouch = 'crouch',
+    Action = 'action',
+    Walk = 'walk',
+    ConversationYes = 'conversation_yes',
+    ConversationNo = 'conversation_no',
+    GroupControlForwards = 'group_control_forwards',
+    GroupControlBack = 'group_control_back',
+    EnterExit = 'enter_exit',
+    VehicleFire = 'vehicle_fire',
+    VehicleSecondaryFire = 'vehicle_secondary_fire',
+    VehicleLeft = 'vehicle_left',
+    VehicleRight = 'vehicle_right',
+    SteerForward = 'steer_forward',
+    SteerBack = 'steer_back',
+    Accelerate = 'accelerate',
+    BrakeReverse = 'brake_reverse',
+    RadioNext = 'radio_next',
+    RadioPrevious = 'radio_previous',
+    RadioUserTrackSkip = 'radio_user_track_skip',
+    Horn = 'horn',
+    SubMission = 'sub_mission',
+    Handbrake = 'handbrake',
+    VehicleLookLeft = 'vehicle_look_left',
+    VehicleLookRight = 'vehicle_look_right',
+    VehicleLookBehind = 'vehicle_look_behind',
+    VehicleMouseLook = 'vehicle_mouse_look',
+    SpecialControlLeft = 'special_control_left',
+    SpecialControlRight = 'special_control_right',
+    SpecialControlDown = 'special_control_down',
+    SpecialControlUp = 'special_control_up',
+    EnterExit = 'enter_exit',
+}
+local _animationBlocks = {
```

Comment:
why is the casing different here that in the other enums?

Reply:
Its more readable to create a local variable than to have everything in a static table. And it wont interfere with anything